src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js

```diff
@@ -40,9 +41,26 @@ const ItemAcquisition = ({ accordionId, itemId }) => {
     );
   }
 
-  const { orderLine, order, piece, vendor } = itemAcquisition;
+  const { orderLine, order, piece, vendor, finance, orderSetting } = itemAcquisition;
   const receiptDate = getDateWithTime(piece?.receivedDate);
 
+  const getAmountWithCurrency = (locale, currency, amount = 0) => (
+    Intl.NumberFormat(locale, { style: 'currency', currency }).format(amount)
+  );
+
+  const totalOrderLineLocationsQuantity = orderLine?.locations?.reduce(
+    (sum, location) => sum + (location.quantity ?? 0), 0
+  );
+
+  const totalFinanceEncumbranceAmountExpended = finance?.transactions?.reduce(
+    (sum, transaction) => sum + (transaction.encumbrance?.amountExpended ?? 0), 0
+  );
+
+  const allFundCodes = orderLine?.fundDistribution?.map(fd => fd.code).filter(code => !!code).join(', ') ?? '';
+  const financeCurrency = finance?.transactions?.length ? finance?.transactions[0].currency : undefined;
```

Comment:
This can be simplified as `const financeCurrency = finance?.transactions[0]?.currency;` to work without ternary